pkgs/os-specific/linux/raspberrypi-eeprom/default.nix

```diff
@@ -0,0 +1,47 @@
+{ stdenvNoCC, lib, fetchFromGitHub, makeWrapper
+, python3, binutils-unwrapped, findutils, kmod, pciutils, raspberrypi-tools
+}:
+stdenvNoCC.mkDerivation {
+  pname = "raspberrypi-eeprom";
+  version = "unstable-2020-05-15"; # USB boot on the beta channel
+
+  src = fetchFromGitHub {
+    owner = "raspberrypi";
+    repo = "rpi-eeprom";
+    rev = "ad18a5b468f787ed37ab62e0a699dabeaa580e27";
+    sha256 = "sha256-E+Fq7VO+gLbCVAawuYfYxBFGER6nelgR8EHwSnc37GQ=";
+  };
```

Comment:
I believe we should follow the stable (critical?) channel by default, but most likely we also want to provide a trivial-to-use way to use beta; I'm not sure if it's common to provide an attribute to the derivation like `betaChannel` and change the source on that attribute.

Though, we should follow the stable channel, that is for sure, we can't have a beta update slip on user's systems if they don't want it.

Reply:
the channels on the eeprom are a bit confusing
the latest version for each channel is all in a single git repo, at the same time, with slightly confusing names
i think the one in `critical` on master, is considered stable and safe for use by anybody

Reply:
I see... this could get confusing fast.

I would recommend **not** putting that comment about USB boot, though possibly a comment stating that we are following the tip of master, which does not mean that the firmware files are unstable.

Though, at the current time, it seems using a tagged release is possible, and might be preferred still, if only to better communicate what is being used?

`v2020.05.28-137ad` is basically the tip, except for a small QOL change in the eeprom script.